Tasks/FtpUpload/ftpuploadtask.ts

```diff
@@ -89,6 +91,9 @@ function doWork() {
             if (ftpOptions.clean) {
                 console.log(tl.loc('CleanRemoteDir', ftpOptions.remotePath));
                 await ftpHelper.cleanRemote(ftpOptions.remotePath);
+            } else if (ftpOptions.cleanContents) {
+                console.log(tl.loc('CleanRemoteDirContents', ftpOptions.remotePath));
+                await ftpHelper.cleanRemoteContents(ftpOptions.remotePath);
```

Comment:
Do we really need another method on the helper for this? Could we not pass in a pattern like /path/foo/* that would delete the contents. Or simply add a parameter to cleanRemote?